### PR TITLE
CART-161 added migration for hashing password

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/migrations/scripts/v012.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  script: (next) => {
+    DAL.users.getAllUsersWithPassword((err, resp) => {
+      let length = resp.length;
+      let i = 0;
+      for(; i < length; i++) {
+        if (resp[i].password && !passwordHash.isHashed(resp[i].password)) {
+          resp[i].password = passwordHash.generate(resp[i].password);
+          DAL.users.updateUser(resp[i]._id, resp[i], next);
+        } else {
+          next;
```
Comment (dmitrij-borchuk) [inline dev/migrations/scripts/v012.js]: "next;" ?

File: dev/migrations/scripts/v012.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  version: 12,
+  message: 'Hashing password for users.',
+  script: (next) => {
+    DAL.users.getAllUsersWithPassword((err, resp) => {
+      let length = resp.length;
+      let i = 0;
+      for(; i < length; i++) {
+        if (resp[i].password && !passwordHash.isHashed(resp[i].password)) {
+          resp[i].password = passwordHash.generate(resp[i].password);
+          DAL.users.updateUser(resp[i]._id, resp[i], next);
```
Comment (dmitrij-borchuk) [inline dev/migrations/scripts/v012.js]: "next" should be called only once, not after every user

File: dev/routing/auth.js [hunk 3]
```diff
@@ -31,6 +31,7 @@ module.exports = function (server) {
       handler: function (request, reply) {
         const user = request.payload;
         DAL.users.getUserForLogin(user.login, (err, doc) => {
+          console.log(user.password, doc.password, '--------------11111111111111111');
```
Comment (dmitrij-borchuk) [inline dev/routing/auth.js]: Remove console.log

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/routing/auth.js (hunk 3 above)
Comment (RomanFedytskyi) [reply dev/routing/auth.js]: fixed

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**